.github/CODEOWNERS

```diff
@@ -0,0 +1,101 @@
+#  - each group must have at least one person, but several people are preferable to avoid a single point of failure issues
+#  - most file groups have a global approvers group as fallback:
+#    - @SAP/spartacus-global-owners: for approving minor changes, large-scale refactorings, and emergency situations.
+#  - a small number of file groups can have very limited number of reviewers because incorrect changes to the files they guard would have serious consequences (e.g. security, public api)
+#
+#  Configuration nuances:
+#
+#  - This configuration works in conjunction with the protected branch settings that require all changes to be made via pull requests with at least one approval.
+#  - This approval can come from an appropriate codeowner, or any repo collaborator (person with write access) if the PR is authored by a codeowner.
+#  - Each codeowners team must have write access to the repo, otherwise their reviews won't count.
+
+
+# ================================================
+#  GitHub username registry
+#  (just to make this file easier to understand)
+# ================================================
+
+#  Platonn - Krzysztof Platis
+#  znikola - Nikola Zarić
+#  developpeurweb - Miguel Estrada
+#  psmul - Patryk Smul
+#  plabadie - Patrick Labadie
+#  WeizhengSap - Weizheng Gao
+#  hackergil - Gilberto Alvarado Jimenez
+
+######################################################################################################
+#
+# CODEOWNERS rules
+# -----------------
+#
+# All the following rules are applied in the order specified in this file.
+# The last rule that matches wins!
+#
+# See https://git-scm.com/docs/gitignore#_pattern_format for pattern syntax docs.
+#
+######################################################################################################
+
+# ================================================
+#  Default Owners
+# (in case no pattern matches a path in a PR - this should be treated as a bug and result in adding the path to CODEOWNERS)
+# ================================================
+
+# * @SAP/spartacus-codeowners
+
+# ================================================
+#  Core
+# ================================================
+/projects/core/**                                   @SAP/spartacus-codeowners
+/projects/storefrontlib/**                          @SAP/spartacus-codeowners
+/core-libs/**                                       @SAP/spartacus-codeowners
+/ci-scripts/**                                      @SAP/spartacus-codeowners
+/scripts/**                                         @SAP/spartacus-codeowners
+/tools/**                                           @SAP/spartacus-codeowners
+tsconfig.*                                          @SAP/spartacus-codeowners
+package.json                                        @SAP/spartacus-codeowners
+yarn.lock                                           @SAP/spartacus-codeowners
```

Comment:
Good stuff. 
What do you think about protecting also other global files in the first level of the root directory?
(we have there a lot of shared stuff: angular.json, sonar-project.properties, .travis.yml, extra-webpack.config.js. But also some global files we don't want accidental changes from other teams as well: prettier config, gitignore, etc.)

I guess this should work, but please double check:
```
/*                                           @SAP/spartacus-codeowners
```
(inspired by https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax)

Reply:
Nice. I'll give it a try.